scripts/post-generator.js

```diff
@@ -0,0 +1,31 @@
+const {writeFileSync, readdirSync, readFileSync, lstatSync} = require('fs');
+const {join, basename} = require('path');
+
+const source = `${process.cwd()}/documentation-site/pages/blog/`;
```

Comment:
this would depend on where you run the script from - should we rather `path.resolve(__dirname, '../../documentation-site/pages/blog')`?